server/src/socobouser/handlers/SocoboUserRoles.handler.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from "express";
+import { IDatabase } from "pg-promise/typescript/pg-promise";
+import { DbExtensions } from "../../app/index";
+import { SocoboUserRole } from "../models/SocoboUserRole";
+
+export class SocoboUserRolesHandler {
+  private _db: IDatabase<DbExtensions>&DbExtensions;
+
+  constructor (db: any) {
+    this._db = db;
+  }
+
+  public getAll = (req: Request, res: Response): void => {
+    this._db.socobouserRoles.getAll()
+      .then((result: SocoboUserRole[]) => res.status(200).json(result))
+      .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
+  }
+
+  public getById = (req: Request, res: Response): void => {
+    const roleId: number = +req.params.id;
+    this._db.socobouserRoles.getById(roleId)
+      .then((result: SocoboUserRole) => res.status(200).json(result))
+      .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
+  }
+
+  public updateById = (req: Request, res: Response): void => {
```

Comment:
I think we will define the roles once at development time (an can extend the roles if we update the app). I think the admin should maybe able to give or remove a role from a user but he should not be able to create new roles. What do you think.